kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/KafkaProxyFrontendHandler.java

```diff
@@ -155,19 +318,22 @@ State state() {
     }
 
     public void outboundChannelActive(ChannelHandlerContext ctx) {
+        this.outboundCtx = ctx;
+    }
+
+    public void outboundChannelUsable() {
```

Comment:
private

Reply:
If it did ever need to be more visible than private it doesn't any longer. Its now private.